salt/formulas.sls

```diff
@@ -1,68 +1,70 @@
-{% set processed_gitdirs = {} %}
-{% set processed_gitdir_envs = [] %}
-{% set processed_basedirs = [] %}
+{%- set processed_gitdirs = {} %}
+{%- set processed_gitdir_envs = [] %}
+{%- set processed_basedirs = [] %}
 
-{% from "salt/map.jinja" import formulas_settings with context %}
-{% from "salt/formulas.jinja" import formulas_git_opt with context %}
-{% from "salt/formulas.jinja" import formulas_opts_for_git_latest with context %}
+{%- from "salt/map.jinja" import formulas_settings with context %}
+{%- from "salt/formulas.jinja" import formulas_git_opt with context %}
+{%- from "salt/formulas.jinja" import formulas_opts_for_git_latest with context %}
 
 # Loop over all formulas listed in pillar data
-{% for env, entries in salt['pillar.get']('salt_formulas:list', {}).items() %}
-{% for entry in entries %}
+{%- for env, entries in salt['pillar.get']('salt_formulas:list', {}).items() %}
+{%-   for entry in entries %}
 
-{% set basedir = formulas_git_opt(env, 'basedir')|load_yaml %}
-{% set gitdir = '{0}/{1}'.format(basedir, entry) %}
-{% set update = formulas_git_opt(env, 'update')|load_yaml %}
+{%-     set basedir = formulas_git_opt(env, 'basedir')|load_yaml %}
+{%-     set gitdir = '{0}/{1}'.format(basedir, entry) %}
+{%-     set update = formulas_git_opt(env, 'update')|load_yaml %}
 
-{% if formulas_settings.checkout_orig_branch %}
-{% if not salt['file.directory_exists']('{0}/{1}'.format(gitdir, '.git')) %}
-{% set gitdir_branch = '' %}
-{% else %}
-{% set gitdir_branch = salt['git.current_branch'](gitdir) %}
-{% endif %}
-{% do processed_gitdirs.update({gitdir:gitdir_branch}) %}
-{% endif %}
+{%-     if formulas_settings.checkout_orig_branch %}
+{%-       if not salt['file.directory_exists']('{0}/{1}'.format(gitdir, '.git')) %}
+{%-         set gitdir_branch = '' %}
+{%-       else %}
+{%-         set gitdir_branch = salt['git.current_branch'](gitdir) %}
+{%-       endif %}
+{%-       do processed_gitdirs.update({gitdir:gitdir_branch}) %}
+{%-     endif %}
 
 # Setup the directory hosting the Git repository
-{% if basedir not in processed_basedirs %}
-{% do processed_basedirs.append(basedir) %}
+{%-     if basedir not in processed_basedirs %}
+{%-       do processed_basedirs.append(basedir) %}
 {{ basedir }}:
   file.directory:
-    {%- for key, value in salt['pillar.get']('salt_formulas:basedir_opts',
-                                             {'makedirs': True}).items() %}
+    {%-   for key, value in salt['pillar.get'](
+            'salt_formulas:basedir_opts',
+            {'makedirs': True}
+          ).items() %}
     - {{ key }}: {{ value }}
```

Comment:
@getSurreal I accept your point here.  The fallback here is the line continuation style that you're used to in Python.  You can see mine!  It's a very opinionated topic but I'm fairly relaxed about this and happy to reset here, except for the slight indent of the `for` to match the surrounding Jinja.